store/store.c

```diff
@@ -75,6 +75,25 @@ static const struct StoreOps *StoreOps[] = {
   NULL,
 };
 
+/**
+ * is_store - Is the argument a NeoMutt store (database)
+ * @param name  Command name to be searched for
+ * @retval true  Function found
+ * @retval false Function not found
+ */
+bool is_store(const char *name)
+{
+  const struct StoreOps **ops = StoreOps;
+
+  for (; *ops; ops++)
+  {
+    if (mutt_istr_equal(name, (*ops)->name))
```

Comment:
Why do you want it case-insensitive? When we get the backend from mutt_hcache_backend we expect it to match with stcmp